.github/workflows/bench.yml

```diff
@@ -0,0 +1,102 @@
+name: "Run Benchmark"
+
+on:
+  workflow_dispatch:
+    inputs:
+      bench-file:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      bench-file:
+        required: true
+        type: string
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    # runs-on: self-hosted
+
+    strategy:
+      matrix:
+        node-version: [16.0.0, 16.18.1, 16.19.0, 16.20.0, 18.0.0, 18.14.2, 18.15.0, 18.16.0, 19.0.0, 19.8.0, 19.8.1, 19.9.0, 20.0.0]
+      fail-fast: true
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js v${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install
+        run: npm install    
+
+      - name: Run Benchmark
+        id: benchmark
+        run: node ${{ inputs.bench-file }} > ./bench-result.md
+
+      - name: Append Machine Info to Benchmark Result
+        run: node scripts/machine-info.mjs >> ./bench-result.md
+
+      - name: Set Env Variable
+        run: |
+          result=$(cat ./bench-result.md)
+          echo "$result" >> $GITHUB_STEP_SUMMARY
+          export RESULT_IN_BASE64=$(cat ./bench-result.md | base64 -w 0)
+          echo "BENCH_RESULT=$RESULT_IN_BASE64" >> "$GITHUB_ENV"
```

Comment:
I think it's better to run it in your fork first. I believe BASE64 will skip \n\t\r